fe/fe-core/src/main/java/org/apache/doris/persist/TableInfo.java

```diff
@@ -131,7 +170,14 @@ public void readFields(DataInput in) throws IOException {
         partitionId = in.readLong();
 
         newTableName = Text.readString(in);
+        oldTableName = Text.readString(in);
```

Comment:
The read order of fields is important. Since the default serialize/deserialize way of `TableInfo` has changed to GSON, after VERSION 134, there is no need to read and set fields in the `readFields` method.

Reply:
done